runtime/service.go

```diff
@@ -497,10 +494,16 @@ func (s *service) CreateVM(requestCtx context.Context, request *proto.CreateVMRe
 }
 
 func (s *service) publishVMStart() error {
+	if s.eventExchange == nil {
```

Comment:
This function will be panicked without this check. Do we want to have an error instead?